src/Api.Rest.Dtos/Data/InspectionPlanDtoBase.cs

```diff
@@ -32,9 +33,9 @@ public abstract class InspectionPlanDtoBase : IAttributeItemDto
 
 		private IReadOnlyList<AttributeDto> _Attributes;
 
-		[JsonProperty( "path" ), JsonConverter( typeof( PathInformationConverter ) )]
+		[Newtonsoft.Json.JsonProperty( "path" ), Newtonsoft.Json.JsonConverter( typeof( PathInformationConverter ) )]		

```

Comment:
Here and in the next line there are some unnecessary indentations.

Reply:
Fixed